### PR TITLE
Topic/error views

### PR DESCRIPTION
**Pull request by chaochaowu**
Customize error pages as 404 and 500
1. plus errors routes
2. modified ipman error pages

### CODE REVIEW

File: app/views/errors/404.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>IpMan: 你的需求我沒辦法幫你...</h1>
```
Comment (chain329) [inline app/views/errors/404.html.erb]: i18n

File: app/views/errors/500.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<h1>IpMan: Oops! 你要不要稍後再試試看？</h1>
```
Comment (chain329) [inline app/views/errors/500.html.erb]: i18n

File: app/views/errors/404.html.erb (hunk 1 above)
Comment (chaochaowu) [reply app/views/errors/404.html.erb]: 已修正xdd

File: app/views/errors/500.html.erb (hunk 2 above)
Comment (chaochaowu) [reply app/views/errors/500.html.erb]: 已修正xdd

**Pull request merged**